src/canvas.py

```diff
@@ -19,8 +19,13 @@ class Canvas(FigureCanvas):
     """Create the graph widget"""
     def __init__(self, parent):
         self.parent = parent
-        style_path = plot_styles.get_user_styles(
-            self.parent)[self.parent.plot_settings.plot_style]
+        try:
+            style_path = plot_styles.get_user_styles(
+                self.parent)[self.parent.plot_settings.plot_style]
+        except KeyError:
+            self.parent.plot_settings.plot_style = "adwaita"
```

Comment:
maybe we shouldn't hardcode adwaita but use the first availabe style from get_user_styles()

Reply:
I think either works, but using the first available style is a bit less maintenance-prone as it doesn't have to be changed upon name changes. Changed it.